app/validators/claim/advocate_claim_sub_model_validator.rb

```diff
@@ -4,29 +4,20 @@ def has_one_association_names_for_steps
       case_details: [],
       defendants: [],
       offence_details: [],
-      fees: %i[
-        assessment
-        certification
-      ]
+      additional_information: []
     }
   end
 
   def has_many_association_names_for_steps
     {
       case_details: [],
-      defendants: [
-        :defendants
-      ],
+      defendants: %i[defendants],
       offence_details: [],
-      fees: %i[
-        basic_fees
-        misc_fees
-        fixed_fees
-        expenses
-        messages
```

Comment:
📝 ⚠️  I've tested this several times, tracked which controllers and validators where taking care of `messages`, `redeterminations` and could not see this validator being used at all for those models.